VAMobile/src/utils/appointments.tsx

```diff
@@ -533,7 +533,7 @@ export const getPastAppointmentDateRange = (): AppointmentsDateRange => {
 
   return {
     startDate: threeMonthsEarlier.startOf('day').toISO(),
-    endDate: todaysDate.minus({ days: 1 }).endOf('day').toISO(),
+    endDate: todaysDate.endOf('day').toISO(),
```

Comment:
There is technically a bug with losing some upcoming appointments that are in the grace period in the last few hours of the day. For example, if I have an appointment at 11:30PM of Aug 1 and I use the app at 12:15AM of Aug 2, the appointment will not show up in either upcoming appointments nor past appointments after all the retrievals and filtering has taken place. This is not specific to mobile and I think this is also a bug in the web tools. Then again, I don't know how many people have appointments in the last hour of the day (or telehealth appointments in the last 4 hours of the day). I think we should file a ticket to track this investigation but it's probably lower priority since we haven't heard of any feedback for this specific case.